e2e/integration/account.test.js

```diff
@@ -5,6 +5,7 @@ it("shows the user's details", () => {
 	cy.findByRole("link", { name: /account/i }).click();
 	cy.findByText(email).should("exist");
 	cy.findByText("Cyf Admin").should("exist");
+	cy.findByRole("heading", { level: 1 }).should("contain.text", "Resources");
```

Comment:
❓ This change seems unrelated to these changes

❌ There's no E2E test that _is_ related to these changes